### PR TITLE
Updated integrationDbOps.js with more detailed setup logic

### PR DESCRIPTION
**Pull request by Nana-EC**
**Detailed description**:
REST API tests have been hard to decipher, modify and are sometimes not consistent.
These changes provide some scaffolding and changes to improve the integration tests on the REST side

-Updated files to camelCase
- Updated integrationDbOps files to contain spec setup steps

**Which issue(s) this PR fixes**:
Partially addresses #412 with inspiration from #529

**Special notes for your reviewer**:
Next PR will update spec files and make current tests use integrationDbOps setup methods

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: hedera-mirror-rest/__tests__/integration.test.js [hunk 1]
```diff
@@ -326,8 +326,19 @@ fs.readdirSync(specPath).forEach(function(file) {
...
   test(`DB integration test - ${file} - ${spec.url}`, async () => {
+    // await specSetupSteps(spec);
     let response = await request(server).get(spec.url);
     expect(response.status).toEqual(spec.responseStatus);
     expect(JSON.parse(response.text)).toEqual(spec.responseJson);
   });
 });
+
+const specSetupSteps = async function(spec) {
+  let recordFile = spec.fileName;
```
Comment (apeksharma) [inline hedera-mirror-rest/__tests__/integration.test.js]: earlier, many tests were setting up record file due to foreign key restriction in t_transactions.fk_rec_file_id. Now the restriction is gone.
If that's the only reason for it to be here, consider removing it in future.

File: hedera-mirror-rest/__tests__/integrationDbOps.js [hunk 2]
```diff
@@ -189,11 +268,29 @@ const toAccount = function(str) {
 };
 
 const addAccount = async function(account) {
+  account = Object.assign({entity_shard: 0, entity_realm: 0, exp_time_ns: null}, account);
+
+  let e = accountEntityIds[account.entity_num];
```
Comment (apeksharma) [inline hedera-mirror-rest/__tests__/integrationDbOps.js]: if there's an existing test case where we add duplicate accounts for testing some corner case, this is okay.
If we don't have a reason to accept duplicates here, we should fail explicitly.
okay in followup.

File: hedera-mirror-rest/__tests__/integration.test.js (hunk 1 above)
Comment (Nana-EC) [reply hedera-mirror-rest/__tests__/integration.test.js]: Gotcha will look into this.

File: hedera-mirror-rest/__tests__/integrationDbOps.js (hunk 2 above)
Comment (Nana-EC) [reply hedera-mirror-rest/__tests__/integrationDbOps.js]: Will look into it

**Pull request merged**